### DISCUSSION TITLE
Question: Game Mode

### DISCUSSION THREAD

**Discussion by Yug1900** [rocksdanister/lively#523] [General]
**Question: Game Mode**

I Have a question: What is this game mode thing, what changes?

**Comment by rocksdanister** [comment]
It only checks for `QUNS_RUNNING_D3D_FULL_SCREEN` state for pausing wallpapers:
https://docs.microsoft.com/en-us/windows/win32/api/shellapi/ne-shellapi-query_user_notification_state

**Comment by sounddrill31** [comment]
Is there a way I can add them manually? Like rainmeter does. It helps me out a lot

**Reply by rocksdanister** [reply]
Just use the default foreground pause algorithm and add application rules.
